vcluster/_fragments/integrations/cert-manager.mdx

```diff
@@ -0,0 +1,197 @@
+Create and apply Issuer and Certificate
+
+After the virtual cluster is running, create an Issuer and Certificate inside the virtual cluster. This guide uses a `letsencrypt-staging` issuer for demonstration purposes.
+
+</Step>
+
+<Step>
+Create the Certificate
+
+Create a file named `certificate.yaml` with the following content:
+
+```yaml title="Certificate configuration"
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: quickstart-example-tls
+  namespace: default
+spec:
+  dnsNames:
+  - example.example.com
+  issuerRef:
+    group: cert-manager.io
+    kind: Issuer
+    name: letsencrypt-staging
+  secretName: quickstart-example-tls
+  usages:
+  - digital signature
+  - key encipherment
+```
+</Step>
+<Step>
+Apply the Certificate configuration:
+
+```bash title="Apply the Certificate configuration"
+kubectl apply -f certificate.yaml
+```
+
+:::tip
+After the certificate is created in the virtual cluster, the integration syncs the created secret back to the virtual cluster after the cert-manager operator creates it in the host cluster.
+:::
+
+</Step>
+</Flow>
+
+## Verify the integration
+
+<Flow id="verify-integration">
+  <Step>
+Check the status of the Issuer:
+
+   ```bash title="Check Issuer status"
+   kubectl describe clusterissuer letsencrypt-staging
+   ```
+  </Step>
+
+  <Step>
+Check the status of the Certificate:
+
+   ```bash title="Check Certificate status"
+   kubectl describe certificate quickstart-example-tls
+   ```
+  </Step>
+  <Step>
+Verify that the secret containing the certificate has been created:
+
+   ```bash title="Verify secret creation"
+   kubectl get secret quickstart-example-tls
+   ```
+  </Step>
+</Flow>
+
+
+## Using the certificate in an application
+
+To use the created certificate in an application, reference the secret in your Ingress resource:
+
+```yaml title="Example Ingress using the certificate"
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: example-ingress
+  annotations:
+    kubernetes.io/ingress.class: nginx
+spec:
+  tls:
+  - hosts:
+    - example.example.com
+    secretName: quickstart-example-tls
+  rules:
+  - host: example.example.com
+    http:
+      paths:
+      - path: /
+        pathType: Prefix
+        backend:
+          service:
+            name: example-service
+            port:
+              number: 80
+```
+
+## Troubleshooting
+
+If you encounter issues with the cert-manager integration, consider the following:
+
+- Ensure that cert-manager is properly installed and running in the host cluster.
+- Check the cert-manager logs in the host cluster for any error messages.
+- Verify that the Issuer and Certificate resources are correctly configured.
+- Ensure that the virtual cluster has the necessary permissions to create and manage certificates.
+
+For more detailed troubleshooting, refer to the [cert-manager troubleshooting guide](https://cert-manager.io/docs/troubleshooting/).
+
+## Config reference
+
+<CertManagerPartial />
```

Comment:
<img width="377" alt="Screenshot 2024-12-16 at 1 18 30 PM" src="https://github.com/user-attachments/assets/ea9ddf47-dbfb-4efb-bccd-0b6c9897be73" />

What's the point of this?

Reply:
This is a placeholder for the config will be generated. It doesn't exist yet.